tox.ini

```diff
@@ -105,5 +105,5 @@ deps =
 commands =
     charm: pyright --pythonversion 3.8 {[vars]src_path}
     lib: pyright --pythonversion 3.8 {[vars]lib_path}
-    lib: /usr/bin/env sh -c 'for m in $(git diff main --name-only {[vars]lib_path}); do if ! git diff main $m | grep -q "+LIBPATCH\|+LIBAPI"; then echo "You forgot to bump the version on $m!"; exit 1; fi; done'
+    lib: /usr/bin/env sh -c 'for m in $(git diff main --name-only {[vars]lib_path}); do if git ls-tree -r HEAD --name-only | grep -q "^$m$"; then  if ! git diff main $m | grep -q "+LIBPATCH\|+LIBAPI"; then echo "You forgot to bump the version on $m!"; exit 1; fi; fi; done'
```

Comment:
this check fails since it sees a change in the library file v1, although the change is that it got deleted. So, I added a check to not error out if the file has been deleted in the branch.